hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/HoodieTableSource.java

```diff
@@ -188,9 +188,11 @@ public DataStream<RowData> produceDataStream(StreamExecutionEnvironment execEnv)
           InputFormat<RowData, ?> inputFormat = getInputFormat(true);
           OneInputStreamOperatorFactory<MergeOnReadInputSplit, RowData> factory = StreamReadOperator.factory((MergeOnReadInputFormat) inputFormat);
           SingleOutputStreamOperator<RowData> source = execEnv.addSource(monitoringFunction, getSourceOperatorName("split_monitor"))
+              .uid("uid_split_monitor_" + conf.getString(FlinkOptions.TABLE_NAME))
               .setParallelism(1)
               .keyBy(MergeOnReadInputSplit::getFileId)
               .transform("split_reader", typeInfo, factory)
+              .uid("uid_split_reader_" + conf.getString(FlinkOptions.TABLE_NAME))
               .setParallelism(conf.getInteger(FlinkOptions.READ_TASKS));
           return new DataStreamSource<>(source);
```

Comment:
Can we reuse the util method: `opItentifier` ?

Reply:
Of course, I have optimized it.

Reply:
Can you apply the review patch though ~

Reply:
done.

Reply:
Did you notice that i have pasted a patch under the comments, you can apply the patch with command:
```java
git apply xxx.patch
```

Reply:
OK